extras/UploadToDatabase.R

```diff
@@ -46,5 +46,21 @@ for (i in (1:length(listOfZipFilesToUpload))) {
   )
 }
 
+tablesInResultsDataModel <- CohortDiagnostics::getResultsDataModelSpecifications() |> 
```

Comment:
Remove native pipe. Its use is inconsistent throughout the package, it requires R 4.1.0 (which is NOT a requirement of the shiny app) and its behaviour is different to they tidyverse's pipe in subtle ways that will be confusing.

Reply:
Are you looking at the right code? i dont see any native pipes? I removed them last week